nomad-base/src/settings/mod.rs

```diff
@@ -221,39 +153,47 @@ pub struct Settings {
     /// The path to use for the DB file
```

Comment:
the longterm goal should be to deprecate and remove this type in favor of using the config types more directly

Reply:
Yep I think having a struct `BaseAgentSettings` in `configuration` would be good and then all the types with lots of functionality (e.g. CachingHome) implement `From<BaseAgentSettings>`. Something like that where instead of implementing `into_<some struct>` we implement `from_<some config struct>`

Reply:
https://github.com/nomad-xyz/rust/issues/40